bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/TeaserImpl.java

```diff
@@ -204,11 +205,15 @@ private void populateActions() {
         Resource actionsNode = resource.getChild(Teaser.NN_ACTIONS);
         if (actionsNode != null) {
             for (Resource actionRes : actionsNode.getChildren()) {
-                actions.add(new Action(actionRes));
+                actions.add(new Action(actionRes, this.getId()));
             }
         }
     }
 
+    public String getId() {
```

Comment:
Isn't it inherited from the Component model getId()?

Reply:
`TeaserImpl` extends `AbstractImageDelegatingModel` which does not extend `AbstractComponentImpl`